tutorminio/plugin.py

```diff
@@ -24,6 +24,8 @@
         "VIDEO_UPLOAD_BUCKET_NAME": "openedxvideos",
         "HOST": "files.{{ LMS_HOST }}",
         "CONSOLE_HOST": "minio.{{ LMS_HOST }}",
+        "GRADES_BUCKET_NAME": "openedxgrades",
+        "QUERYSTRING_AUTH": False,
```

Comment:
Since openedxgrades bucket is private so `MINIO_QUERYSTRING_AUTH` should be `true` by default otherwise we will get error when trying to download csv 
<img width="1346" alt="image" src="https://github.com/overhangio/tutor-minio/assets/79192414/f40cad56-1439-434a-a3e9-46323bf10d98">